TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -19,75 +23,85 @@
 @Slf4j
 public class LoggingHttpRequestHandler extends ChannelInboundHandlerAdapter {
     static class SimpleHttpRequestDecoder extends HttpRequestDecoder {
-        /**
-         * Override to broaden the visibility.
-         *
-         * @param ctx    the {@link ChannelHandlerContext} which this {@link ByteToMessageDecoder} belongs to
-         * @param buffer the {@link ByteBuf} from which to read data
-         * @param out    the {@link List} to which decoded messages should be added
-         * @throws Exception
-         */
         @Override
-        public void decode(ChannelHandlerContext ctx, ByteBuf buffer, List<Object> out) throws Exception {
-            super.decode(ctx, buffer, out);
+        public HttpMessage createMessage(String[] initialLine) throws Exception {
```

Comment:
nit: I know this is existing code but may be helpful to callout the benefit of using PassThruHttpHeaders here

Reply:
Done.  Thanks